client/dashboard/sites/settings-caching/index.tsx

```diff
@@ -76,10 +76,17 @@ export default function CachingSettings( { siteSlug }: { siteSlug: string } ) {
 		e.preventDefault();
 		edgeCacheStatusMutation.mutate( formData.active, {
 			onSuccess: () => {
-				createSuccessNotice( __( 'Settings saved.' ), { type: 'snackbar' } );
+				createSuccessNotice(
+					formData.active
+						? __( 'Global edge caching enabled.' )
```

Comment:
The feature is called `Global edge cache` 🤔 can we call it like this as well here?